tcpdevmem.h

```diff
@@ -0,0 +1,15 @@
+#include "thread.h"
+#include "lib.h"
+
+#define PAGE_SHIFT (12)
+#define PAGE_SIZE (1 << PAGE_SHIFT)
+
+#ifndef MSG_SOCK_DEVMEM
+#define MSG_SOCK_DEVMEM 0x2000000	/* don't copy devmem pages but return
+					 * them as cmsg instead */
```

Comment:
Do you need this? Does the header file not define MSG_SOCK_DEVMEM?

Reply:
```
nvcc -arch=sm_90 -O3 -g -I usr/include -D_GNU_SOURCE -DNO_LIBNUMA -DWITH_TCPDEVMEM_CUDA -c -o tcpdevmem_cuda.o tcpdevmem_cuda.cu
tcpdevmem_cuda.cu(313): error: identifier "MSG_SOCK_DEVMEM" is undefined

1 error detected in the compilation of "tcpdevmem_cuda.cu".
make: *** [Makefile:89: tcpdevmem_cuda.o] Error 2
```

Even if I `#include <linux/socket.h>`